NetCoreDemos/Views/Splitter/Overview.cshtml

```diff
@@ -0,0 +1,87 @@
+<div class="dx-viewport">
+    <div class="demo-container">
+        @(Html.DevExtreme().Splitter()
+            .ID("splitter")
+            .Items(items => {
+                items.Add()
+                    .Resizable(true)
+                    .Size("140px")
+                    .MinSize("70px")
+                    .Template(new JS("function(data){return paneContentTemplate(data, 'Left Pane')}"));
+
+                items.Add().Splitter(s => {
+                    s.Orientation(Orientation.Vertical);
+                    s.Items(verticalItems => {
+                        verticalItems.Add()
+                            .Resizable(true)
+                            .Collapsible(true)
+                            .MaxSize("75%")
+                            .Template(new JS("function(data){return paneContentTemplate(data, 'Central Pane')}"));
+
+                        verticalItems.Add()
+                            .Collapsible(true)
+                            .Splitter(horizontalSplitter => {
+                                horizontalSplitter.Orientation(Orientation.Horizontal);
+                                horizontalSplitter.Items(horizontalItems => {
+                                    horizontalItems.Add()
+                                        .Resizable(true)
+                                        .Collapsible(true)
+                                        .Size("30%")
+                                        .MinSize("5%")
+                                        .Template(new JS("function(data){return paneContentTemplate(data, 'Nested Left Pane')}"));
+
+                                    horizontalItems.Add()
+                                        .Collapsible(false)
+                                        .Template(new JS("function(data){return paneContentTemplate(data, 'Nested Central Pane')}"));
+
+                                    horizontalItems.Add()
+                                        .Resizable(true)
+                                        .Collapsible(true)
+                                        .Size("30%")
+                                        .MinSize("5%")
+                                        .Template(new JS("function(data){return paneContentTemplate(data, 'Nested Right Pane')}"));
+                                });
+                            });
+                    });
+                });
+
+                items.Add()
+                    .Resizable(false)
+                    .Size("140px")
+                    .Collapsible(false)
```

Comment:
May be put Collapsible after Resizable like you do it everywhere here...
However, ever without this, it looks good to me